scripts/integration.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
 
-go run main.go instances add dev
 go run main.go serve &
-sleep 3
+sleep 5
+go run main.go instances add dev
+
 cd integration-tests/pouchdb
 npm install
 npm run test
+testresult=$?
```

Comment:
maybe `npm install && npm run test`, to avoid running the tests if the npm install fails